src/configs/strings/th.json

```diff
@@ -379,6 +379,10 @@
     "message": "ระยะท้องที่",
     "description": "column header: for the status of Focus Investigation plans"
   },
+  "FI_COLUMN_STATUS": {
```

Comment:
Added this to avoid modifying the `FI_STATUS` which affects other files and there was a request to change it a while back. See [here](https://github.com/onaio/reveal-frontend/pull/1465/commits/86b99ad6ef1c76ed386067d9564d1b5f5fd8feb3).